src/components/MessageBox/MessageBox.spec.js

```diff
@@ -15,3 +22,26 @@ describe('MessageBox component', () => {
 		expect(okBTN).toBeInTheDocument();
 	});
 });
+
+describe('MessageBox component with type "start"', () => {
+	it('should render children', () => {
```

Comment:
This does not test what you describe what it should test :) `react children` are referred to as nodes which you can call in between your component call:
```jsx
<MyComponent>
  <span>I am a child!</span>
  <span>Me too.</span>
</MyComponent>
```